examples/reference/.env

```diff
@@ -1,5 +1,5 @@
 # Tide API
-CONTENT_API_SERVER=https://develop.content.reference.sdp.vic.gov.au/
+CONTENT_API_SERVER=https://nginx-php.pr-132.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au
```

Comment:
@yeniatencio please revert this change.

The better way to change the CONTENT_API_SERVER value for a PR branch is to use the lagoon CLI tool to change the value, that way you don't need to commit this file. 

See https://digital-vic.atlassian.net/wiki/spaces/DDS/pages/2050949826/How-to+update+an+environment+variable+using+Lagoon

Otherwise if its just for local development just either run the dev command with the value before it eg: `CONTENT_API_SERVER=https://nginx-php.pr-132.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au npm run dev` _OR_ export the value, eg: `export CONTENT_API_SERVER=https://nginx-php.pr-132.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au`